types/react-table.config.d.ts

```diff
@@ -0,0 +1,141 @@
+/**
```

Comment:
Are there no public types for this?

Reply:
Also wondering why `@types/react-table` not enough.

Reply:
The biggest pitfall of `react-table` is separated types. The author doesn't provided them on his own, so even latest `@types/react-table` not always correlate with available codebase. In my specific case some of the types related to `useSortBy` hook was missing so typescript won't allow to compile actually proper code.

Contents inside `react-table.config.d.ts` comes from community fixing the issue.

Reply:
I would consider to try `material-table` instead `react-table`. Wdyt?

Reply:
Local copy of types feels bad. And it also feels bad that `@types/react-table` is not up to date. Did we consider any alternatives?